### PR TITLE
Change decision hover states, center content unless complex

### PR DESCRIPTION
**Pull request by oliverviljamaa**
image

### CODE REVIEW

**Review by Tankenstein — APPROVED** [review]

**Review by OFFLlNE — APPROVED** [review]
image

File: less/decision.less [hunk 1]
```diff
@@ -1,10 +1,16 @@
...
+  padding: 24px 12px 24px 0;
+  margin-bottom: 0 !important;
+
   border-top: 1px solid @brand-smoke-plus-10;
   border-bottom: 1px solid @brand-smoke-plus-10;
-  margin-bottom: 0 !important;
+
+  align-items: center;
+
+  .transition(all ease @decision-focus-transition);
```
Comment (StevePole) [inline less/decision.less]: Does this need to be all?  Can we just do background-color?

File: less/variables.less [hunk 2]
```diff
@@ -1004,6 +1004,7 @@
 @panel-heading-color:         @brand-navy-minus-10;
 @panel-border-radius:         @border-radius-base;
 @panel-focus-transtion:       @transition-duration;
+@decision-focus-transition:   @transition-duration;
```
Comment (StevePole) [inline less/variables.less]: Is this the only decision variable?  It should be grouped with the others if there are any.  If the only one, should anything else be made a variable?

**Review by StevePole — COMMENTED** [review]
Couple questions...

**Review by StevePole — APPROVED** [review]

**Pull request merged**